yarn.lock

```diff
@@ -16298,6 +16408,34 @@ esbuild@^0.15.0:
     esbuild-windows-64 "0.15.18"
     esbuild-windows-arm64 "0.15.18"
 
+esbuild@^0.19.3:
+  version "0.19.3"
+  resolved "https://registry.yarnpkg.com/esbuild/-/esbuild-0.19.3.tgz#d9268cd23358eef9d76146f184e0c55ff8da7bb6"
+  integrity sha512-UlJ1qUUA2jL2nNib1JTSkifQTcYTroFqRjwCFW4QYEKEsixXD5Tik9xML7zh2gTxkYTBKGHNH9y7txMwVyPbjw==
+  optionalDependencies:
+    "@esbuild/android-arm" "0.19.3"
+    "@esbuild/android-arm64" "0.19.3"
+    "@esbuild/android-x64" "0.19.3"
+    "@esbuild/darwin-arm64" "0.19.3"
+    "@esbuild/darwin-x64" "0.19.3"
+    "@esbuild/freebsd-arm64" "0.19.3"
+    "@esbuild/freebsd-x64" "0.19.3"
+    "@esbuild/linux-arm" "0.19.3"
+    "@esbuild/linux-arm64" "0.19.3"
+    "@esbuild/linux-ia32" "0.19.3"
+    "@esbuild/linux-loong64" "0.19.3"
+    "@esbuild/linux-mips64el" "0.19.3"
+    "@esbuild/linux-ppc64" "0.19.3"
+    "@esbuild/linux-riscv64" "0.19.3"
+    "@esbuild/linux-s390x" "0.19.3"
+    "@esbuild/linux-x64" "0.19.3"
+    "@esbuild/netbsd-x64" "0.19.3"
+    "@esbuild/openbsd-x64" "0.19.3"
+    "@esbuild/sunos-x64" "0.19.3"
+    "@esbuild/win32-arm64" "0.19.3"
+    "@esbuild/win32-ia32" "0.19.3"
+    "@esbuild/win32-x64" "0.19.3"
```

Comment:
Looking at the diff here has me wondering if we could just bump `@badeball/cypress-cucumber-preprocessor` to get the required `@esbuild` deps. For context:
```
➜  amplify-ui git:(ui-react-auth/add-rn-clipboard) ✗ npm ls @esbuild/darwin-arm64
amplify-ui@ /Users/cpollman/Amplify/x_backup/amplify-ui
├─┬ @aws-amplify/ui-angular-mono@0.0.1 -> ./packages/angular
│ └─┬ jest-preset-angular@13.1.1
│   └─┬ esbuild@0.17.19
│     └── @esbuild/darwin-arm64@0.17.19
└─┬ @aws-amplify/ui-e2e@0.0.1 -> ./packages/e2e
  └─┬ @badeball/cypress-cucumber-preprocessor@16.0.3
    └─┬ esbuild@0.17.19
      └── @esbuild/darwin-arm64@0.17.19 deduped
```

Reply:
`badeball/cypress-cucumber-preprocessor` >v17 requires node18 or higher ([release note](https://github.com/badeball/cypress-cucumber-preprocessor/releases/tag/v17.0.0)) so that is going to be a no go for now.